### PR TITLE
Updated dockerfile according to new fsl-base

### PR DESCRIPTION
**Pull request by Thorpe7**
Updated dockerfile to reflect changes in base-fsl image. Updated versioning of manifest for fsl-anat version/image.

### CODE REVIEW

**Comment by Thorpe7** [comment]
@pvelasco

**Review by pvelasco — COMMENTED** [review]
It looks good to me. Just change the base image once the new `flywheel/fsl-base:6.0.2` is available.

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # Creates docker container that runs FSL pipelines
-#
-
-# Use prepared fsl-base:6.0.1 based on ubuntu:xenial
-FROM flywheel/fsl-base:6.0.1
+# Use prepared fsl-base:6.0.1 based on ubuntu:focal
+FROM swarm_test/fsl-base:6.0.2
```
Comment (Thorpe7) [inline Dockerfile:3]:
```suggestion
FROM flywheel/fsl-base:6.0.2
```

**Review by Thorpe7 — COMMENTED** [review]
Change to reflect Flywheel's docker hub

**Pull request merged**